modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import {faroConfig} from './faro.config';
+
+export const test = mergeTests(loginAnalyticsCloudTest(), loginTest());
+
+test('renders workspace list', async ({page}) => {
```

Comment:
Hey @marcosapmf, can you change the test name to be more related to the test proposal?